website/src/reference/Methods.tsx

```diff
@@ -63,6 +63,8 @@ function Reference({
 }
 
 function Methods({ methods }: { methods: Method[] }) {
+  console.log(methods);
```

Comment:
lint error - no console statement should go live

Reply:
removed